benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField-WA.json

```diff
@@ -26,4 +26,4 @@
   {"LiveFrom": "08/03/2023", "CaseTypeID": "Benefit", "CaseFieldID": "issueInterlocDecisionDate", "AccessProfile": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
   {"LiveFrom": "08/03/2023", "CaseTypeID": "Benefit", "CaseFieldID": "issueInterlocDecisionDate", "AccessProfile": "caseworker-sscs-superuser", "CRUD": "CRUD"},
   {"LiveFrom": "08/03/2023", "CaseTypeID": "Benefit", "CaseFieldID": "issueInterlocDecisionDate", "AccessProfile": "caseworker-sscs-judge", "CRUD": "CRUD"}
```

Comment:
could we get rid of these whitespace changes so that it doesn't confuse things

Reply:
done